distrobuilder/main_repack-windows.go

```diff
@@ -489,7 +489,7 @@ func (c *cmdRepackWindows) injectDrivers(infDir, driversDir, filerepositoryDir,
 	softwareRegistry := "Windows Registry Editor Version 5.00"
 	for driverName, driverInfo := range windows.Drivers {
 		logger.WithField("driver", driverName).Debug("Injecting driver")
-		infFilename := fmt.Sprintf("oem%d.inf", i)
+		infFilename := fmt.Sprintf("oem-virtio-incus%d.inf", i)
```

Comment:
I don't know if there are any "magical" assumptions about the inf files being named "oemX.inf", but driver installation seemed to continue working properly with the renamed inf files.